compose.example.yml

```diff
@@ -0,0 +1,15 @@
+---
+services:
+  sml2mqtt:
+    image: spacemanspiff2007/sml2mqtt
+    container_name: sml2mqtt
+    init: true
+    mem_limit: 32m
```

Comment:
Does it make sense to include `mem_limit` in the default docker file?

Reply:
I think it's reasonable to give a hint on the expected memory consumption. On my amd64 test machine, the process never went above 40 MB, so I think 64m should be OK. I don't insist on it, just tell if you are concerned.